libsolidity/Types.h

```diff
@@ -407,6 +408,9 @@ class FunctionType: public Type
 	/// Can contain a nullptr in which case indicates absence of documentation
 	ASTPointer<ASTString> getDocumentation() const;
 
+	/// true iff arguments are to be padded to multiples of 32 bytes for external calls
+	bool padArguments() const { return !(m_location == Location::SHA3 || m_location == Location::SHA256 || m_location == Location::RIPEMD160); }
```

Comment:
Small suggestion for predicate functions like this whose name could be considered ambiguous. Maybe change to `shouldPadArguments()` or `canPadArguments()`. If it was lisp we could just have an identifier with a question mark like `padArguments?()`